# -*- coding: utf-8 -*-
"""Covid live data .ipynb

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1peG30st9CoPnfrRn1sFF0PyQmCIa8HCz

! is to run the command from the console
"""

!pip install datadotworld[pandas]

"""configure api from data.world"""

!dw configure

import datadotworld as dw

import pandas as pd

covid_cases = dw.load_dataset('covid-19-data-resource-hub/covid-19-case-counts')

"""Explores what is inside covid_cases"""

covid_cases.dataframes

covid_df = covid_cases.dataframes['covid_19_activity']

"""check the why of the error"""

covid_df.head(10)

"""The error is not affecting what I want to do!"""

covid_df_reduced = covid_df[['report_date',
'continent_name',
'country_short_name',
'province_state_name',
'people_positive_cases_count',
'people_positive_new_cases_count',
'people_death_count',
'people_death_new_count']]

covid_df_reduced.head(10)

from google.colab import files
covid_df_reduced.to_csv('COVID-19 Activity.csv', index=False)
files.download('COVID-19 Activity.csv')

"""here go to the google data studio"""

